tutorials/how-to/_posts/2017-02-15-program.md

```diff
@@ -116,7 +116,7 @@ Let's say you want to add a bird that moves around your garden scene. First, bre
 
 ## Get Help
 
-That might seem like a lot of work, but it will seriously save you a ton of time and headaches if you code this way. It'll also make it a lot easier to get help on [the forum](http://forum.HappyCoding.io) and [Stack Overflow](http://stackoverflow.com/). 
+That might seem like a lot of work, but it will seriously save you a ton of time and headaches if you code this way. It'll also make it a lot easier to get help on [the happyCoding forum](http://forum.HappyCoding.io) and [Stack Overflow](http://stackoverflow.com/). 
```

Comment:
Would you mind breaking this up into two words? "Happy Coding" instead of "happyCoding", that way screen readers announce it properly. Thanks!

Reply:
Done :)